### PR TITLE
0209 update settings to complete rollover

### PR DESCRIPTION
**Pull request by defong**
### Context
Rollover

### Changes proposed in this pull request
Rollovered

### Guidance to review
Best to view by commits
Config change cause some broken tests due to hardcoring.
As well as the ethereal nature of recruitment cycle vs allocation year over allocation

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by defong** [comment]
[Update settings to complete rollover and set current cycle to 2021 (API)](https://trello.com/c/xtODRKEf/209-update-settings-to-complete-rollover-and-set-current-cycle-to-2021-api)

**Comment by defong** [comment]
[When cycle ends, move the "Next cycle" create PE forms to "current cycle"](https://trello.com/c/kLNgJpjs/211-when-cycle-ends-move-the-next-cycle-create-pe-forms-to-current-cycle)

**Review by gpeng — APPROVED** [review]

File: spec/requests/api/v2/allocations_spec.rb [hunk 1]
```diff
@@ -1,6 +1,17 @@
 require "rails_helper"
 
 RSpec.describe "/api/v2/allocations/<id>", type: :request do
+  before do
+    # Note: the concept of recruitment cycle here is ethereal.
+    # it is tied to a recruitment cycle that is associated to
+    # Settings.allocation_cycle_year
+    allocation_recruitment_cycle
```
Comment (misaka) [inline spec/requests/api/v2/allocations_spec.rb:8]: In the settings we call this `allocation cycle`. That makes more sense than `allocation recruitment cycle`, in my mind at least, should we use that or is the longer version correct?

**Review by misaka — APPROVED** [review]
Approving as we're short on time.

**Pull request merged**